app/controllers/api/host_aggregates_controller.rb

```diff
@@ -25,6 +25,32 @@ def delete_resource_main_action(_type, host_aggregate, _data = {})
       {:task_id => host_aggregate.delete_aggregate_queue(current_user.userid)}
     end
 
+    def add_host_resource(type, id, data = {})
+      raise BadRequestError, "Must specify an id" if id.nil? || data["host_id"].nil?
```

Comment:
Still think that it would be helpful to have a separate check for the host_id so that the error that we raise indicates which was missing (e.g. in this suggestion https://github.com/ManageIQ/manageiq-api/pull/1124#discussion_r806178047)

Reply:
Looks like this one is still missing the other raise

Reply:
```suggestion
      raise BadRequestError, "Must specify an id" if id.nil?
      raise BadRequestError, "Must specify a host_id" if data["host_id"].nil?
```